src/components/main-navigation-links.tsx

```diff
@@ -26,7 +25,7 @@ const mainNavigationLinks = ({
     aria-controls={`${ ariakey }-menu`}
   >
     <Lang lnkey={lnkey}/>
-    <MdArrowDropDown onClick={preventOpenLink}/>
+    <MdArrowDropDown />
```

Comment:
please test this on mobile, I suspect it will be broken.

Reply:
@Idrinth everything okay for this pull request? please let me know if I need to modify or check something else